stubs/docker/docker/api/network.pyi

```diff
@@ -9,16 +9,16 @@ class NetworkApiMixin:
         self,
         name: str,
         driver: str | None = None,
-        options: dict[Incomplete, Incomplete] | None = None,
+        options: dict[str, Any] | None = None,
```

Comment:
From https://docs.docker.com/network/drivers/bridge/#options I see that the option keys are strings, and the values vary.